### DISCUSSION TITLE
Does handheld companion work with glosSI

### DISCUSSION THREAD

**Discussion by Spidy123222** [Valkirie/HandheldCompanion#697] [General]
I was looking into getting handheld companion but I notice it also does virtual controller stuff. Glossi works with steams steam input, games with launchers and the steam overlay in big picture mode like steamOS 3 for all inputs which steams driver handles. So it made me wonder if it would support it. From looking at videos it could conflict

**Comment by CasperH2O** [comment]
@Spidy123222 there is currently no one on the HC development team who has experience with a SteamDeck Windows, HC and Glossi. I recommend you ask on the HC Discord SteamDeck channel or SteamDeck Discord Windows channel.
